reemgawad/brunchtime
# Directions page

What Has changed :

- Created page for "Directions" (after you click on "take me there").
- Implemented standard map ( I've created a Trello card to implement the nice getdirection feature one).
- implemented "I have arrived button" that update the visit to arrived:true and redirect to feedback(visit) edit page.
- implemented 'back to brunch places button" that brings you back to the index page.

How to test :

- Click on Take me there on restaurant and make sure you are redirected to the new page with the map
- If you click on "I have arrived" you should be prompted to edit the rating 
- Once updated you should be able to see the above rating on restaurant page
